docs/usage/oauth.md

```diff
@@ -47,13 +56,7 @@ http.createServer(onRequest).listen(3000);
 
 ```ts
 app.get('/login', async (req, res) => {
-  let authRoute = await Shopify.Auth.beginAuth(
-    req,
-    res,
-    SHOP,
-    '/auth/callback',
-    true,
-  );
+  let authRoute = await Shopify.Auth.beginAuth(req, res, SHOP, '/auth/callback');
```

Comment:
It seems w're defaulting this example to online and the other above to offline - we should make sure they're consistent.